lib/src/pub.dart

```diff
@@ -87,6 +108,41 @@ class Pubspec {
       this._contents,
       this._yamlMap);
 
+  static Future<Pubspec> init() {
+    var completer = new Completer();
```

Comment:
I always implement Future-returning methods with something like:

`=> new Future(() { /* code */ })`

to ensure all errors are caught, and for readability (clear that a Future is actually returned).